shared: Fix RHEL8 os_variant

ldoktor opened the pull request:
Couple of fixes to RHEL8 kickstart/config and some unification with Fedora/RHEL7.

luckyh commented on shared/cfg/guest-os/Linux/RHEL/8.cfg:
```diff
@@ -2,13 +2,15 @@
     variants:
         - aarch64:
             vm_arch_name = aarch64
+        - ppc64:
```
Seems that RHEL-8 does not have ppc64 builds.

ldoktor replied:
Yes and I don't insist on this commit. It'd just make my setup simpler.

The long story is that I'm using a single guest-os profile in order to keep using just a single vm image, be it RHEL7 or RHEL8, because I simply don't have enough disk space and I'm running el7, el7a and el8 checks within a single job. So this commit would help me in using RHEL8 profile for RHEL7 variants as well, but I agree it's ugly. If you say you want me to remove it, I will not object and will have to adjust my CI.

ldoktor commented:
Hello @luckyh, I removed the controversial commit with ppc64(be) and fixed my setup to not to require it. It was not working anyway as oldish RHEL7 libvirt doesn't know rhel8.0 profile... No changes to the other commits.

luckyh commented:
> Hello @luckyh, I removed the controversial commit with ppc64(be) and fixed my setup to not to require it. It was not working anyway as oldish RHEL7 libvirt doesn't know rhel8.0 profile... No changes to the other commits.

ok, thanks!

xutian commented on shared/unattended/RHEL-8-devel.ks:
```diff
@@ -1,5 +1,6 @@
 install
 KVM_TEST_MEDIUM
+GRAPHICAL_OR_TEXT
 text
```
this line should be drop, right?

ldoktor replied:
Oh, true.

ldoktor commented:
@xutian I removed the extra `text` entry in `RHEL-8-devel.ks` and rebased it.

ldoktor commented:
@xutian would you please review the changes?

xutian commented:
ACK
